lesson04/index.js

```diff
@@ -0,0 +1,11 @@
+function numberSquarer() {
+    let input = prompt('Enter a Number to Square');
+    reult = 0;
```

Comment:
To initialize a variable, you'll want to first start it like this:

**let result = 0;**

On your line 3 of index.js, your result is miss-spelt and not given the **let** command.